src/main/java/local/alfonso/countries/CountryCompareAge.java

```diff
@@ -0,0 +1,11 @@
+package local.alfonso.countries;
+
+import java.util.Comparator;
+
+public class CountryCompareAge implements Comparator<Country> {
+    @Override
+    public int compare(Country c1, Country c2)
+    {
+        return c1.getMedAge().compareTo(c2.getMedAge());
+    }
```

Comment:
Line 9 - since you are dealing with integers, you can just subtract `c1.getMedAge()` by `c2.getMedAge()` (doing so will produce the same result as a `String` comparison anyway.